modules/custom/d_social_media/content/blocks/d_social_media_example_header.yml

```diff
@@ -0,0 +1,12 @@
+info:
+  type: block_plugin
+  plugin: 'social_media_block'
+  langcode: en
+  theme: droopler_subtheme
```

Comment:
to nam troche utrudni zadanie, bo niedlugo bedzie albo droopler_subtheme albo drupalorg_subtheme i przywiązywaie bedzie takie ze zalezy kto skad sciagnie drooplera to taka defaultowa skroke bedzie mial wlaczona

Reply:
To moze niech będzie opcja typu "current" albo "default"

Reply:
to może to theme tutaj niepotrzebne w ogole? może by to obejsc i dodawac zawsze do tego "aktualnego" ?

and one question, why are you not using english here? :thinking: